services/user/justfile

```diff
@@ -0,0 +1,3 @@
+
+proto:
+    python -m grpc_tools.protoc -I. --python_out=. --pyi_out=. --grpc_python_out=. user.proto
```

Comment:
Maybe add a clean files for convenience

Reply:
They're `.gitignored`, when you run the command again they will be replacedtoo. I don't see a reason to clean them as you _will_ need them to do anything.